event/event.go

```diff
@@ -0,0 +1,181 @@
+// event is a package that models events that occur during the execution of a program
+package event
+
+// Event is an interface that custom events should implement.
+// It is empty for now because there are no general methods
+// required of events yet.
+type Event interface {
+}
+
+// KeyCharacter is an event that occurs when a character is actually typed on
+// the keyboard. This may be provided by an input method.
+type KeyCharacter struct {
+	// Code of the key pressed or released
+	Code int
+	// Key board modifiers
+	Modifiers int
```

Comment:
ditto

Reply:
OK, I will do this (see previous comment).

Reply:
OK!